react/features/transcription/components/TranscriptionLabel.web.js

```diff
@@ -0,0 +1,145 @@
+import JitsiMeetJS from '../../base/lib-jitsi-meet';
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { shouldRemoteVideosBeVisible } from '../../filmstrip';
```

Comment:
If you focus on the invite button pr, https://github.com/jitsi/jitsi-meet/pull/1950, then this additional positioning logic for remote videos can be removed.